mikupad.html

```diff
@@ -1514,6 +1593,24 @@
 		</div>`;
 }
 
+function onLoad() {
+	// Any other general task we want to do when the page is loaded.
+	setTimeout(() => {
+		try {
+			document.getElementById("prompt-container").style.maxWidth = "36em";
+			// You can't actually get the width from the element unless you do some
+			// stupid workaround like "get the computed width in pixels, get the
+			// font size, do math on pixels, and convert to em". Instead, we'll just
+			// manually set it to be "36em" so it can be interacted with by the modal.
+
+			// document.getElementsByClassName("modal-width-readout")[0].value = "36";
+		} catch(e) {
+			reportError(e);
+		}
+	}, 1000);
+}

```

Comment:
This feels wrong, you should do this with React's useEffect instead, and also you should load the value from the persistent state.

Reply:
It is indeed a stupid hack. I can make a go at using React to do this more elegantly than just plain JS.